src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -34,29 +35,6 @@ public static Builder builder() {
     return new AutoValue_Candidate.Builder();
   }
 
-  // creates a new Candidate object by extracting the properties from "obj"
-  public static Candidate fromJSONObject(JSONObject obj) throws JSONException {
```

Comment:
Sorry I think I should have asked this earlier.. is there a reason why we're renaming the method from "fromJSONObject" to "fromVoterInfoQuery"?  

Same for Contest#fromVoterInfoQuery and Election#fromElectionQuery.

Reply:
I did that because Election now has fromElectionQuery AND fromVoterInfoQuery, and I wanted to keep it consistent across the other objects (esp. once we add in other APIs that will also be JSON objects filling in separate fields)